prototype1/src/components/MetricSelect.tsx

```diff
@@ -0,0 +1,23 @@
+import { Metric } from "../metrics"
+import {useId} from '@react-aria/utils'
+
+interface MetricSelectProps {
+  onChange: (metric: Metric) => void
+}
+
+export function MetricSelect(props: MetricSelectProps) {
+  let id = useId();
+  return (
+    <div className="stack">
+      <label className="option-text" htmlFor={id}>Color metric</label>
+      <select
+        id={id}
+        onChange={(event) => props.onChange(event.target.value as Metric)}
+      >
+        {Object.entries(Metric).map(([key,value]) => (
```

Comment:
Key not used

Reply:
resolved in [3fc5b35](https://github.com/git-visual/git-visual/pull/119/commits/3fc5b350b9053511c8f103f2e4f9ec4861c8d9f3)